draft-dnsop-deleg.md

```diff
@@ -263,53 +232,67 @@ Some zone owners may wish to use multiple providers to serve their zone, in whic
 
 \[ DRAFT NOTE: SVCB says that there "SHOULD only have a single RR". This ignores that but keeps the randomization part. Section 2.4.2 of SVCB \]
 
-### Loop Prevention
+## Loop Prevention
 
 The TargetName of an SVCB or DELEG record MAY be the owner of a CNAME record. Resolvers MUST follow CNAMEs as well as further alias SVCB records as normal, but MUST not allow more then 4 total lookups per delegation, with the first one being the DELEG referral and then 3 SVCB/CNAME lookups maximal.
 
 Special care should be taken by both the zone owner and the delegated zone operator to ensure that a lookup loop is not created by having two AliasMode records rely on each other to serve the zone. Doing so may result in a resolution loop, and likely a denial of service. The mechanism on following CNAME and SVCB alias above should prevent exhaustion of server resources. If a resolution can not be found after 4 lookups the server should reply with a SERVFAIL error code.
 
-## Deployment Considerations
-
-The DELEG and SVCB records are intended to replace the NS record while also adding additional functionality in order to support additional transports for the DNS. Below are discussions of considerations for deployment.
+#  Examples
 
-### AliasMode and ServiceMode in the Parent
+To introduce DELEG record, this example shows the authority section of a DNS response that delegates a subdomain to another nameserver.
 
-Both the AliasMode and ServiceMode records can be returned for the DELEG record from the parent. This is different from the SCVB  {{?RFC9460}} specification and only applies for the DELEG RRSet in the parent.
 
+    example.com.  86400  IN DELEG  1 ns1.example.com. (
+                    ipv4hint=192.0.2.1 ipv6hint=2001:DB8::1 )
+    example.com.  86400  IN NS     ns1.example.com.
+    ns1.example.com.    86400   IN  A  192.0.2.1
+    ns1.example.com     86400   IN  AAAA    2001:DB8::1
 
-### Rollout
+In this example, the authoritative nameserver is delegating using the same parameters as regular DNS, but the delegation as well as the glue can be signed.
 
-When introduced, the DELEG and SVCB records might not initially be supported by the DNS root or TLD operators. Zone owners may place these records into their zones before the zones above them have done so. However, doing so is only useful for further delegations down the tree as an SVCB record at the zone apex alone does not indicate a new delegation type. The only way to discover new delegations is with the DELEG record at the parent.
+Like in SVCB, DELEG also offer the ability to use the Alias form of delegation. The example below shows an example where example.com is being delegated with a DELEG AliasMode record which can then be further resolved using standard SVCB to locate the actual parameters.
```

Comment:
Perhaps "Like with SVCB, DELEG offers the ability to use the Alias form of delegation."

Reply:
Thanks Tim. Did changed the text.